app/api/auth.py

```diff
@@ -67,31 +66,27 @@ def login():
        payload_user_login = {'username': username, "password": password, "action": "login", "token": None}
        response_user_token = requests.post(url=URL_login, json=payload_user_login)
        token = response_user_token.json()
-        
-        #Api for fetching all user data from hr api.
-       user = get_current_user()
-       if user["role"] == "Admin":
-            URL_all_details = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
-            payload_all_user_details = {"action": "get_enable_user","token":token['data']['token']}
-            response_all_user_details = requests.post(url=URL_all_details, json=payload_all_user_details)
-            result = response_all_user_details.json()
-            user = mongo.db.all_users.insert({
-            "All_users": result
-            })
-       else:
-           pass
-        
-        
+         
        if token['data'] == {'message': 'Invalid Login'}:
            return jsonify(msg='invalid login')
        else:
-           URL_details = 'https://hr.excellencetechnologies.in/attendance/sal_info/api.php'
+           URL_details = 'http://dev.hr.excellencetechnologies.in/hr/attendance/sal_info/api.php'

```

Comment:
base url should come from config

Reply:
but if we add this url in config.py then there will be two urls